src/hotspot/share/runtime/arguments.cpp

```diff
@@ -892,74 +892,69 @@ static bool set_bool_flag(JVMFlag* flag, bool value, JVMFlagOrigin origin) {
   }
 }
 
-static bool set_fp_numeric_flag(JVMFlag* flag, char* value, JVMFlagOrigin origin) {
+static bool set_fp_numeric_flag(JVMFlag* flag, const char* value, JVMFlagOrigin origin) {
+  if (*value == '\0' || isspace(*value)) {
```

Comment:
Please preceded with comment

// strtod allows leading whitespace, but our flag format does not.

Reply:
Fixed.

Reply:
Sorry I meant for the comment to precede line 896.